demisto_sdk/commands/common/constants.py

```diff
@@ -878,15 +879,16 @@ def urljoin(*args: str):
     return reduce(lambda a, b: str(a).rstrip('/') + '/' + str(b).lstrip('/'), args).rstrip("/")
 
 
-class GithubCredentials:
-    ENV_TOKEN_NAME = 'DEMISTO_SDK_GITHUB_TOKEN'
+class GitCredentials:
+    ENV_GITHUB_TOKEN_NAME = 'DEMISTO_SDK_GITHUB_TOKEN'
+    ENV_GITLAB_TOKEN_NAME = 'DEMISTO_SDK_GITLAB_TOKEN'
     TOKEN: Optional[str]
 
     def __init__(self):
-        self.TOKEN = os.getenv(self.ENV_TOKEN_NAME)
+        self.TOKEN = os.getenv(self.ENV_GITHUB_TOKEN_NAME) or os.getenv(self.ENV_GITLAB_TOKEN_NAME)
```

Comment:
I am not sure this is the best behaviour. Think of users who are working with both GitHub and GitLab repos, this will make then need to delete GitHub token everytime they want to access the GitLab repo. Perhaps better to split to 2 tokens.